BUILD.md

```diff
@@ -25,6 +25,15 @@ Build
 
 NOTE: Debugging a CUDA program is not straightforward in Visual Studio, so you cannot do the debugging similar to other applications.
 
+Cli-Build (Using Powershell)
+1. Clone the [GitHub repository](https://github.com/BrownBiomechanics/Autoscoper).
+2. Enter the repos directory
+3. Make and enter a build directory `mkdir build` and `cd build`
+4. Configure project `cmake .. `
+5. Build external dependencies `cmake --build . `
```

Comment:
```
cmake --build . --config Release
```

Reply:
It required to specify the configuration when using multi-config generator like Visual Studio